modules/sfp_bevigil.py

```diff
@@ -0,0 +1,159 @@
+# -*- coding: utf-8 -*-
+# -------------------------------------------------------------------------------
+# Name:         sfp_bevigil
+# Purpose:      Search BeVigil OSINT API for subdomains.
+#
+# Author:      alt-glitch <https://github.com/alt-glitch>
+#
+# Created:     2022-08-27
+# Copyright:   (c) alt-glitch 2022
+# Licence:     MIT
+# -------------------------------------------------------------------------------
+
+import json
+import time
+
+from spiderfoot import SpiderFootEvent, SpiderFootPlugin
+
+
+class sfp_bevigil(SpiderFootPlugin):
+    meta = {
+        'name': "BeVigil API",
+        'summary': "Get Subdomains and URLs belonging to one target domain using OSINT Data made available by BeVigil",
```

Comment:
This says URLs are obtained, but the module only returns hosts.

Reply:
Updated!